### ISSUE TITLE
FR: Check that cases aren't duplicated in switch statement

### ISSUE THREAD

**Issue by erikvullings** [palantir/tslint#2916]
When creating a new case in a switch statement, it may happen that I duplicate an existing case. Wouldn't it be nice if this would generate an error (or at least a warning)

```ts
const a = create();
switch (a) {
  case 1:
     ...
     break;
  case 2:
     ...
     break;
  case 1:  // THIS SHOULD GENERATE AN ERROR
     ...
     break;
  default:
     ...
     break;

```

**Comment by JoshuaKGoldberg** [comment]
Oh, this was fixed by #2937! 🎉
